install/advanced/project/index.rst

```diff
@@ -144,16 +152,16 @@ Make an instance out of the ``Django Template``
 
   # Install the Python packages
   cd /opt/geonode_custom/my_geonode
-  pip install -r requirements.txt --upgrade --no-cache --no-cache-dir
-  pip install -e . --upgrade
+  pip install -r src/requirements.txt --upgrade --sno-cache-dir
```

Comment:
@jkariukidev what is this option? I guess you cut the original params by error

Reply:
Was introduced during refactoring by accident. I have corrected it.